common/build/eslint-config-fluid/package.json

```diff
@@ -19,9 +19,10 @@
 		"lint:fix": "npm run prettier:fix && npm run eslint:fix",
 		"prettier": "prettier --check .",
 		"prettier:fix": "prettier --write .",
-		"print-config": "concurrently npm:print-config:default npm:print-config:test npm:print-config:minimal npm:print-config:recommended npm:print-config:strict && npm run cleanup-printed-configs",
+		"print-config": "concurrently npm:print-config:* && npm run cleanup-printed-configs",
```

Comment:
Nice, I didn't know this syntax worked! 😍

Reply:
I figured I would give it a shot and was pleasantly surprised 🙂